### DISCUSSION TITLE
BinanceFuturesMarkPriceUpdate()

### DISCUSSION THREAD

**Discussion by Alpha23-Ltd** [nautechsystems/nautilus_trader#1059] [Q&A] [answered]
Hi, 

Binance Futures have the concept of a mark price for the instruments. 
Is this supported as a data update from the Adapter?

I have tried to subscribe to it in case it comes through the on_data() method, but no luck :

`self.subscribe_data(data_type=DataType(BinanceFuturesMarkPriceUpdate))`

`def on_data(self, data: Data):`
`    if data.data_type == DataType(BinanceFuturesMarkPriceUpdate):`
`    #do something`

thanks in advance

**Comment by cjdsellers** [comment, marked as answer]
Hi @avince 

This isn't yet handled in a first class way in the platform, however an example of how to setup this exact usecase is provided in the docs
https://docs.nautilustrader.io/integrations/binance.html#binancefuturesmarkpriceupdate

Hope that helps!

**Reply by Alpha23-Ltd** [reply]
Awesome. I missed that part of the docs.
Thanks again
